extensions/security/pom.xml

```diff
@@ -48,6 +48,7 @@
     <modules>
         <module>activedirectory</module>
         <module>iprange</module>
+        <module>jwt</module>
```

Comment:
I am not sure if we want to activate it by default. Especially when you mention there being a lot of TODOs open.
Might be wise to make it opt-in.

Also there is an open PR to remove iprange